sorc/orog_mask_tools.fd/inland.fd/nb.F90

```diff
@@ -23,18 +24,20 @@ MODULE cs_nb
   INTEGER :: cres, xres, yres
  
 CONTAINS
-
-!   _______1_______
-!  |               |       1-upper, 2-bottom, 3-left, 4-right 
-!  |               |
-!  |               |
-! 3|               |4
-!  |               |
-!  |               |
-!  |_______________|
-!          2
-!      Figure 1. Boundary numbers
-!
+  
+  !> Initialize inter-panel neighbor index for global grid
+  !! @param[in] cubed sphere resolution (48, 96...)
+  !
```

Comment:
Use the <pre> </pre> tags around any text you want included in the docs. Like this:

```
!! <pre>
!!                 ------------------------------
!!                 some fancy formatting
!! </pre>
```